static/bidder-info/conversant.yaml

```diff
@@ -15,4 +15,4 @@ userSync:
   redirect:
     url: "https://prebid-match.dotomi.com/match/bounce/current?version=1&networkId=72582&rurl={{.RedirectURL}}"
     userMacro: ""
-    # conversant appends the user id to end of the redirect url and does not utilize a macro
+    # epsilon appends the user id to end of the redirect url and does not utilize a macro

```

Comment:
this info is related to `conversant`. Any specific reason to change it to `epsilon`